bitnami/external-dns/templates/psp.yaml

```
@@ -3,7 +3,7 @@
 apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
-  name: {{ template "external-dns.fullname" . }}
+  name: {{ printf "%s-%s" (include "common.names.fullname.namespace" .) "psp" }}
```

You should be using the other form

```suggestion
  name: {{ template "common.names.fullname.namespace" . }}
```

---

Same comment as above.

---

Now that I realised the mistake I made, I fixed it.
I added the suffix where it wasn't needed.